src/bot.py

```diff
@@ -0,0 +1,13 @@
+from discord.ext.commands import Bot, CommandError, Context, MissingPermissions
+
+
+class TeamBot(Bot):
+    @staticmethod
+    async def on_ready() -> None:
+        """Runs when the bot is connected."""
+        print('Awaiting...')
+        print("Bot Is Ready For Commands")
```

Comment:
Not sure this should be a `staticmethod`, and messages should probably be logged rather than printed.

Reply:
Good point, although this PR won't be changing this.